### DISCUSSION TITLE
Welcome to Intelligent-Traffic-Management-System-Using-ML-YOLO Discussions!

### DISCUSSION THREAD

**Discussion by 17110009-HICET-IT** [FYP-ITMS/Intelligent-Traffic-Management-System-using-Machine-Learning#7] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by 17110013-HICET-IT** [comment]
Hey people. Get ready for review today 😎

**Comment by jyothichalla23** [comment]
Hello sir!...
Always getting an error while running this code like this:
RuntimeError: shape '[256, 128, 3, 3]' is invalid for input of size 243099
what the solution for this?
